src/Components/Article/ArticlePage.jsx

```diff
@@ -15,7 +14,11 @@ const ArticlePage = () => {
   return (
     <div>
```

Comment:
Is this div the most appropriate element? You're rendering several items in a list format, would a ol or ul be more appropriate?